src/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics.cpp

```diff
@@ -58,7 +58,7 @@ void Dynamics::DynamicalEquations(
     const Instant& anInstant
 )
 {
-    for (Index i = 0; i < dxdt.size(); ++i)
+    for (Index i = 0; i < (Size)dxdt.size(); ++i)
```

Comment:
`Index` and `Size` are both typedef-ed to size_t, and since we're doing a `.size()` I figured it's clearer to typecast to `Size`. But I'm fine with either, what do you think is clearer from a reader persepective?

Reply:
Yeah true. Totally no functional difference. Perhaps
```suggestion
    for (Size i = 0; i < (Size)dxdt.size(); ++i)
```
then? 

Just a visual quirk IMO